src/generator/parser.ts

```diff
@@ -30,7 +30,7 @@ export const parseODataMetadataFromRemote = async (
   headers,
   fetch
 ) => {
-  const res = await fetch(uri, { headers, })
+  const res = await fetch(uri, { ...headers, })
```

Comment:
Yes, I've reproduced this issue in my site. 

Recommend fix it by modify `src/generator/cli` like this

```js
const meta = await parseODataMetadataFromRemote(
   options.uri,
   GetAuthorizationPair(options.user, options.pass),
   fetch,
)
```

Because this issue is caused by wrong external usage.

Thanks

Reply:
Sure, next commit will do it